MANIFESTO.md

```diff
@@ -10,7 +10,7 @@ While software as a means of production has been and is being increasingly colle
 We are still at the beginning of a new movement, which is becoming increasingly important as the overall complexity of software stacks increases. For true self-determination in the ever-growing digital realm, we urgently need to build skills and share the gained knowledge freely and in unlimited ways also in the realm of operational knowledge — exactly the way we started with software code almost 40 years ago. Let’s start closing the gap to have open source style collaboration not just for the Dev piece in DevOps. And this is just another urgently needed piece to enhance and grow the overall digital capabilities for achieving sovereignty in the digital realm.
 
 ## We build a community of practice
-[...]
+For this to happen we need to bring people together and build a community of practice to establish a low entry barrier to foster a prosperous environment for comfortable exchange. From time to time, we're mentally blocked and may need a sparring partner to think outside of the box. We encourage organizations to build internal and external communities of practice to jointly solve these challenges. Regardless of the individual level of expertise, everyone contributes and enriches with their knowledge and individual views. Communities of practice need to take place both as small tandems or at large scale. Therefore we build a network and connect on different channels that serve asynchronous as well as synchronous communication needs.
```

Comment:
I would suggest to write every paragraph without any reference (*For this to happen...*) to previous content so that every paragraph can be quoted on its own. What do you think?

Reply:
> From time to time, we can get mentally stuck and may need a sparring partner to think outside the box. Open Operations builds a community of practice to keep the barrier to entry low and create a thriving environment for comfortable exchange. Regardless of individual knowledge level, everyone contributes and enriches this community with their knowledge and individual views. Communities of practice connect through various channels serving both asynchronous and synchronous communication needs in the form of small tandems or on a larger scale to solve these challenges together. Therefore, we are building a network by encouraging organizations to foster and connect internal and external communities of practice.

my try on rewording the paragraph as I felt it needed a different flow (while all the bullet points were there already)